src/GeneralStateTestsFiller/stMemoryTest/oogFiller.yml

```diff
@@ -19,9 +19,12 @@ oog:
     0000000000000000000000000000000000010020:
       balance: '0x0ba1a9ce0ba1a9ce'
       code: |
-       :yul {
-          pop(keccak256(0,0x1000))
-       }    
+        :yul berlin
```

Comment:
use lllc.

Reply:
I prefer to use Yul because I think it would make it easier for future maintainers.